AppCreationScripts/Configure.ps1

```diff
@@ -199,7 +199,7 @@ Function ConfigureApplications
    # Add Required Resources Access (from 'service' to 'Microsoft Graph')
    Write-Host "Getting access from 'service' to 'Microsoft Graph'"
    $requiredPermissions = GetRequiredPermissions -applicationDisplayName "Microsoft Graph" `
-                                                -requiredDelegatedPermissions "User.Read";
+                                                -requiredDelegatedPermissions "User.Read" `
```

Comment:
I don't think we need the trailing ` here as there isn't a next line?

Reply:
It gives a syntax error to me, I've also provided a PR for the sample generator for this. Please check this for me. Thanks